woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2123,14 +2132,13 @@ def test_setitem_overwrite_column(sample_df):
         semantic_tags='test_tag',
     )
 
-    match = 'Standard tags have been added to "full_name"'
-    with pytest.warns(StandardTagsChangedWarning, match=match):
-        df.ww['full_name'] = new_series
+    df.ww['full_name'] = new_series
 
     assert 'full_name' in df.columns
     assert 'full_name' in df.ww._schema.columns.keys()
     assert df.ww['full_name'].ww.logical_type == Double
-    assert df.ww['full_name'].ww.semantic_tags == {'test_tag', 'numeric'}
+    assert df.ww['full_name'].ww.use_standard_tags is False
+    assert df.ww['full_name'].ww.semantic_tags == {'test_tag'}
```

Comment:
Note the change in behavior here:

The dataframe was initialized with `use_standard_tags=True`, but when we use `setitem` with a Woodwork column where we set `use_standard_tags=False`, we're no longer updating the column to match the rest of the table because each column is independent, so the semantic tags don't change.